src/www/system_advanced_misc.php

```diff
@@ -221,8 +228,8 @@ function thermal_modules()
     }
 ?>
       <section class="col-xs-12">
-        <form method="post" name="iform" id="iform">
```

Comment:
looks like this clashes with a newer commit on master, reverting the style update